src/app/context/AuthContext.tsx

```diff
@@ -27,6 +27,24 @@ interface AuthContextType {
 const AuthContext = createContext({} as AuthContextType);
 
 export function AuthProvider({ children }: { children: React.ReactNode }) {
+  //theme
+  useEffect(() => {
+    api.getSettings().then((response) => {
```

Comment:
Does it make sense to move this into a i.e. util function because it looks like the same code is being used in 3 places?

Reply:
I thought about what I could do to make this more DRY, but didn't know you could turn a `useEffect` hook into a function. It is possible?

Reply:
Not sure, I assumed you can use a function in that place.

Reply:
Ok I have never done that before but i'll give it a try and see if it works.

Reply:
Looks like I was able to get it working, see latest commit.